src/runtime/runtime.ts

```diff
@@ -64,7 +68,28 @@ export class Runtime {
     return new Runtime(new Loader(), FakeSlotComposer, context);
   }
 
-  constructor(loader?: Loader, composerClass?: new () => SlotComposer, context?: Manifest) {
+  static init(root?: string, urls?: {}) {
+    const map = {...Runtime.mapFromRootPath(root), ...urls};
+    const loader = new Loader(map);
+    const pecFactory = pecIndustry(loader);
+    const runtime = new Runtime(loader, SlotComposer);
+    runtime.pecFactory = pecFactory;
```

Comment:
All of the created objects are local to this function, so without looking at any side effects the various constructors might have it looks like `init` doesn't do anything. If we are relying on non-local side effects, a comment describing them would be helpful.

Reply:
Ok, will add.